### DISCUSSION TITLE
Hyper-V Graphics Resolution

### DISCUSSION THREAD

**Discussion by simonjen1** [Qonfused/OSX-Hyper-V#36] [Q&A] [answered]
Hi,

Any idea when the limitation of 1024x768 graphics limitation will be fixed with Hyper-V?

Thanks

Simon

**Comment by Qonfused** [comment, marked as answer]
There's https://github.com/Qonfused/OSX-Hyper-V/issues/6#issuecomment-2037277344 explaining the main roadblocks in the current graphics bridge implementation, though there is currently nothing planned. However, there are workarounds with virtual displays (e.g. BetterDisplay) which may be worth investigating depending on your use-case.
